trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.ts

```diff
@@ -130,6 +134,10 @@ describe(page, () => {
           /Received payments are entered against reported fees. When payments show as matched, the adjustment data for keying into ACBS will be automatically generated./,
         );
 
+      wrapper
+        .expectText(`[data-cy="how-to-create-record-correction-request-text"]`)
+        .toMatch(/Errors with a fee record can be addressed and queried with the bank through a record correction request./);
+
```

Comment:
Should we add a component test for the feature flag visibility?

Reply:
done